ml-package-versions.yml

```diff
@@ -73,7 +73,8 @@ tensorflow-2.x:
   package_info:
     pip_release: "tensorflow"
     install_dev: |
-      pip install tf-nightly
+      # https://github.com/tensorflow/tensorflow/issues/46429
```

Comment:
@harupy Can we file an internal ticket to track unpinning this once https://github.com/tensorflow/tensorflow/issues/46429 is addressed?